packages/components/src/dialogs/ConfirmationDialog.tsx

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import { ReactElement, ReactNode } from 'react';
+import type { SpectrumLabelableProps } from '@react-types/shared';
```

Comment:
We may want to include @react-types/shared in the exclusion rule, but I haven't thought through it.

Reply:
It creates a circular dependency in some cases. I tried removing it from `@deephaven/react-hooks` and re-exporting from `@deephaven/components`. But `@deephaven/components` already relies on `@deephaven/react-hooks`, so it creates a circular dependency

Reply:
Talked to @mofojed on this. I think this will be fine as-is.